doc/sphinx/source/project/release-notes.rst

```diff
@@ -3,17 +3,22 @@
 Release Notes
 #############
 
-
-
 |version|
-|release|
 
 For downloads formatted as ``.zip`` and ``tar.gz``, see https://github.com/NervanaSystems/ngraph/releases; 
 
 .. important:: Pre-releases (``-rc-0.*``) have newer features, and are less stable.  
 
-CHANGELOG |release|
-===================
+
+Changelog 
+=========
+
+Stable / Verified
+
+0.19
+----
+
+**Verified:** `0.19.0-rc2`_
```

Comment:
`Verified` is a GitHub-ism that AFAICT means no more to us than "Scott happened to used the GitHub UI to do this release". Should probably omit it here.

I think we should keep the `.` between `rc` and `2`, because that is the branch name (and technically the correct way to name it, according to semantic versioning rules).

Reply:
Thanks for the clarification.  There was some confusion on us linking to a previous rc that hadn't gone through all of the validation team's tests (? might ask @aslepko ) and the discussion on slack made me think we wanted some differentiator to distinguish those that had passed our validation checks.

Reply:
Modified that the links point to a download rather than something "verified".